app/javascript/utils/form/final_form/MultiSelectField.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+import { Field } from 'react-final-form'
+
+import MultiSelect from 'components/molecules/MultiSelect'
+
+import { FieldError, HelpText, Label } from './Field'
+
+const format = (options) => {
+  return options && options.map((option) => (typeof option === 'string' ? option : option.value))
+}
+const HEIGHT_NORMAL = 'normal'
+const HEIGHT_SMALL = 'small'
+
+const MultiSelectField = ({
+  blockNote,
+  disabled = false,
+  fieldName,
+  height = HEIGHT_NORMAL,
+  helpText,
+  id,
+  label,
+  options,
+  onChange = () => {}
+}) => (
+  <Field name={fieldName} component={MultiSelect} format={format} formatOnBlur>
+    {({ input, meta }) => {
+      return (
+        <>
+          <div className={classNames('form-group', { error: meta.error && meta.touched })}>
+            <Label
+              label={label}
+              id={id || `form-${fieldName}`}
+              fieldName={fieldName}
+              blockNote={blockNote}
+            />
+            <MultiSelect
+              {...input}
+              options={options}
+              height={height}
+              value={input.value}
```

Comment:
I'm seeing this in the console during QA:
<img width="1257" alt="- 2020-12-02 at 1 35 13 PM" src="https://user-images.githubusercontent.com/64036574/100934178-3b0db600-34a3-11eb-960a-ca0b40f5c3f6.png">

I think `input.value` needs to be converted to an array of SelectItemShape before it's passed into the MultiSelect component